test/App.test.js

```diff
@@ -44,6 +37,10 @@ describe('should fetch data', () => {
 
     return instance.refreshResults(url).then(() => {
       expect(component.state('data')).toEqual('expected');
+      afterEach(() => {
+        mock.reset();
```

Comment:
Is `reset` needed when there is `restore` right after?

Reply:
You're right :) I removed it.